unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioSource.cs

```diff
@@ -162,11 +164,15 @@ private void ApplyLoadedAudioClip(DCLAudioClip clip)
             }
 
             Model model = (Model) this.model;
-            if (audioSource.enabled && model.playing && !audioSource.isPlaying)
+            bool shouldPlay = playedAtTimestamp != model.playedAtTimestamp ||
+                              (model.playing == true && audioSource.isPlaying == false);
+
+            if (audioSource.enabled && model.playing && shouldPlay)
             {
                 //To remove a pesky and quite unlikely warning when the audiosource is out of scenebounds
                 audioSource.Play();
             }
```

Comment:
Our coude convention for one liner if is like that ->
```
if (audioSource.enabled && model.playing && shouldPlay)
    audioSource.Play();
```

Reply:
> Good job! I only left a small comment!

I think is more readable because of the comment. If I remove the comment, I could put like you said. Should I do that? :thinking:

Reply:
Nah, left it that way then!